oximeter/db/src/configs/keeper_config.xml

```diff
@@ -9,13 +9,14 @@
     </logger>
 
     <!-- To allow all use :: -->
-    <listen_host from_env="CH_LISTEN_ADDR"/> 
+    <listen_host from_env="CH_LISTEN_ADDR"/>
     <path replace="true" from_env="CH_DATASTORE"/>
 
     <keeper_server>
         <tcp_port from_env="CH_LISTEN_PORT"/>
         <server_id from_env="CH_KEEPER_ID_CURRENT"/>
         <log_storage_path from_env="CH_LOG_STORAGE_PATH"/>
+        <storage_path from_env="CH_STORAGE_PATH"/>
```

Comment:
I can't find any documentation for this new setting, have you? What does it refer to?

Reply:
I meant to remove that, it was a red herring when I was troubleshooting earlier.
If it isn't explicitly set, it just gets set to the parent directory of `<log_storage_path/>` - it seems to be a way of just defining a parent directory for all of the other pieces, but doesn't seem to be documented.
I expect the tests will pass without it.